src/app/modules/backbone/pages/backbone/backbone.page.ts

```diff
@@ -1,14 +1,42 @@
-import { Component } from "@angular/core";
+import { Component, OnDestroy } from "@angular/core";
+import { ActivatedRoute, Params, Router } from "@angular/router";
+import { Subscription } from "rxjs";
+
+import { ModalService } from "@core/views/modal.service";
+
+import { VolumeSelectionModalComponent } from "@shared/components/modals";
 
 @Component({
     selector: "app-backbone",
     templateUrl: "./backbone.page.html",
     styleUrls: ["./backbone.page.scss"]
 })
-export class BackbonePageComponent {
+export class BackbonePageComponent implements OnDestroy {
     showSidebar: boolean = false;
+    private subscriptions: Subscription = new Subscription();
+
+    constructor(
+        private activatedRoute: ActivatedRoute,
+        private router: Router,
+        private modalService: ModalService<VolumeSelectionModalComponent>,
+    ) {
+        this.subscriptions.add(
+            this.activatedRoute.queryParams.subscribe(
```

Comment:
if done on component init and not on subsequent changes, can use snapshot.queryParams

Reply:
done